simvue_fds/connector.py

```diff
@@ -599,10 +780,39 @@ def launch(
             else self._chid
         )
 
+        if self.slice_parse_quantity:
+            # This is only necessary because of the way pyfdstools works
+            if (
+                self.workdir_path
+                and (
+                    pathlib.Path(self.fds_input_file_path).absolute()
+                    != pathlib.Path(self.workdir_path)
+                    .joinpath(f"{self._chid}.fds")
+                    .absolute()
+                )
+            ) or (
+                not self.workdir_path
+                and (
+                    pathlib.Path(self.fds_input_file_path).absolute()
+                    != pathlib.Path.cwd().joinpath(f"{self._chid}.fds").absolute()
+                )
+            ):
+                shutil.copy(self.fds_input_file_path, f"{self._results_prefix}.fds")
+
+            # Make sure xyz is enabled
+            if not nml["dump"]["write_xyz"]:
```

Comment:
I leave it up to you but my paranoid self would do:
```python
if not nml.get("dump", {}).get("write_xyz"):
```
Just in case for some unlikely reason both keys are missing/corrupted